rpc/src/v1/impls/chain.rs

```diff
@@ -239,6 +239,29 @@ where
         Ok(self.client.block(&BlockId::Hash(block_hash)).map(|block| block.transactions_count()))
     }
 
+    fn get_min_transaction_fee(&self, action_type: String) -> Result<Option<u64>> {
+        let common_parameters = self.client.common_params();
+        Ok(match action_type.as_str() {
+            "MintAsset" => Some(common_parameters.min_asset_mint_cost),
```

Comment:
I think it's better to use camelCase.
Because the action type of RPC uses camelCase.

Reply:
I changed them to `camelCase`s